Makefile.pre.in

```diff
@@ -1431,14 +1431,9 @@ regen-opcode:
 	# using Tools/build/generate_opcode_h.py
```

Comment:
I can't add a comment there, but this also needs some updates in the `regen-cases` target.